src/docs/asciidoc/common/cloud-services/standard-user-workflow/quick-start.adoc

```diff
@@ -1,2 +1,192 @@
 :relative-path: ../../../
-include::{docdir}/variables.adoc[]
\ No newline at end of file
+include::{docdir}/variables.adoc[]
+
+In this part we will explain how to use the **Standard User Workflow** in the nominal case, i.e. using the default implementations of each parts of the workflow (For example, the default implementation to send the confirmation email use the _Mailjet_ API).
+
+Use the Standard User Workflow can be done in 3 parts :
+
+* Use the Standard User Workflow in your worker
+* Add properties to configure the Standard User Workflow
+* Create the client part
+
+== Bootstrap
+
+In this Quickstart, we will begin with a generated application using the CLI. So you can run :
+
+[source, console]
+----
+$ npm init @zetapush myApp
+----
+
+We also use the package `@zetapush/user-management` so you can install it with :
+
+[source, console]
+----
+$ npm install --save @zetapush/user-management
+----
+
+== Worker part
+
+Your worker contains just this code :
+
+.worker/index.ts
+[source,typescript,linenums]
+----
+import { Module } from '@zetapush/core';
+import { StandardUserManagementModule, StandardUserWorkflow, ConfirmationUrlHttpHandler } from '@zetapush/user-management';
+
+@Module({
+  imports: [StandardUserManagementModule], <1>
+  expose: {
+    user: StandardUserWorkflow, <2>
+    http: ConfirmationUrlHttpHandler <3>
+  }
+})
+export default class Api {}
+----
+<1> First you need to export a class (named _Api_ here) that is a module (configured with the _@Module()_ annotation). You import the **StandardUserManagementModule** to use it into your application.
+<2> To call, from the front side, the Standard User Workflow cloud functions, you need to expose it (here in the namespace named _user_).
```

Comment:
The `@zetapush/user-management` library/cloud service provides the StandardUserWorkflow class. The front needs to call methods of the StandardUserWorkflow class that's why StandardUserWorkflow class is exposed. In this example, the StandardUserWorflow methods are exposed in `user` namespace.